### PR TITLE
faq: add entry that explains SSH crypto change in F33

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -336,3 +336,44 @@ systemd:
 
 For more information see
 https://github.com/coreos/fedora-coreos-tracker/issues/519[the tracker issue discussion].
+
+== After the rebase to Fedora 33 why does my tooling that uses SSH cease to work?
```
Comment (jlebon) [inline modules/ROOT/pages/faq.adoc:340]: How about something simpler like

```suggestion
== Why does SSH stop working after upgrading to Fedora 33?
```

?

File: modules/ROOT/pages/faq.adoc [hunk 2]
```diff
@@ -336,3 +336,44 @@ systemd:
...
+and disabling the use of the `ssh-rsa` public key signature algorithm.
+
+You may hit issues if you use RSA keys and:
+
+* use an old version of the `SSH` client
+* use tooling/software libraries that don't support using RSA SHA2 host key signatures
+
+For example, GoLang has an https://github.com/golang/go/issues/37278[open issue]
+to solve this problem in the GoLang stack, but has yet to resolve the
+issue. This has been hit and worked around by the FCOS community in
```
Comment (jlebon) [inline modules/ROOT/pages/faq.adoc:355]:
```suggestion
For example, Go has an https://github.com/golang/go/issues/37278[open issue]
to solve this problem in its SSH implementation, but has yet to resolve
it. This has been hit and worked around by the FCOS community in
```

?

File: modules/ROOT/pages/faq.adoc [hunk 3]
```diff
@@ -336,3 +336,44 @@ systemd:
...
+our build tooling and also our higher level projects:
+
+- https://github.com/coreos/fedora-coreos-tracker/issues/699[coreos/fedora-coreos-tracker#699]
+- https://github.com/coreos/coreos-assembler/issues/1772[coreos/coreos-assembler#1772]
+
+If you run into this problem and need to workaround the issue you
```
Comment (jlebon) [inline modules/ROOT/pages/faq.adoc:361]:
```suggestion
If you run into this problem and need to work around the issue, you
```

File: modules/ROOT/pages/faq.adoc [hunk 4]
```diff
@@ -336,3 +336,44 @@ systemd:
...
+have a few options:
+
+- Switch to a newer non-RSA key type.
```
Comment (jlebon) [inline modules/ROOT/pages/faq.adoc:364]:
```suggestion
- Switch to a newer non-RSA SHA-1 key type.
```

(RSA SHA-2 works, right?)

**Review by jlebon — APPROVED** [review]
Some minor suggestions, but LGTM makes sense overall!

File: modules/ROOT/pages/faq.adoc (hunk 4 above)
Comment (dustymabe) [reply modules/ROOT/pages/faq.adoc:364]: The RSA SHA2 keyex algo works, but using a non RSA key itself will workaround the problem entirely, which is what I'm trying to say here.

**Comment by dustymabe** [comment]
Implemented most of your suggested changes! Thanks for the review.

File: modules/ROOT/pages/faq.adoc [hunk 5]
```diff
@@ -336,3 +336,44 @@ systemd:
...
+In Fedora 33 there was a change to
+https://www.fedoraproject.org/wiki/Changes/StrongCryptoSettings2[implement stronger crypto defaults].
+Part of this included taking the
+https://www.openssh.com/txt/release-8.3[advice of OpenSSH] upstream
+and disabling the use of the `ssh-rsa` public key signature algorithm.
+
+You may hit issues if you use RSA keys and:
+
+* use an old version of the `SSH` client
+* use tooling/software libraries that don't support using RSA SHA2 host key signatures
```
Comment (bgilbert) [inline modules/ROOT/pages/faq.adoc:351]: "public key signatures" would be clearer, since host keys aren't involved here.

**Review by bgilbert — COMMENTED** [review]
LGTM generally.

File: modules/ROOT/pages/faq.adoc (hunk 5 above)
Comment (dustymabe) [reply modules/ROOT/pages/faq.adoc:351]: thanks for the suggestion. Fix in #222

**Pull request merged**